budgetportal/templates/budgetportal/guide_index_page.html

```diff
@@ -30,15 +30,15 @@ <h1 class="Page-mainHeading">{{ page.title }}</h1>
       {{ page.specific.intro|richtext }}
     </div>
 
-{#    <div class="Grid has-twoColumn">#}
-{#      {% for category_guide in guides %}#}
-{#        {% if category_guide.external_url %}#}
-{#          {% include 'components/Preview/index.html' with title=category_guide.external_url_title excerpt=category_guide.external_url_description url=category_guide.external_url %}#}
-{#        {% else %}#}
-{#          {% include 'components/Preview/index.html' with title=category_guide.title excerpt=category_guide.search_description url=category_guide.url %}#}
-{#        {% endif %}#}
-{#      {% endfor %}#}
-{#    </div>#}
+    <div class="Grid has-twoColumn">
+      {% for category_guide in guides %}
+        {% if category_guide.external_url %}
+          {% include 'components/Preview/index.html' with title=category_guide.external_url_title excerpt=category_guide.external_url_description url=category_guide.external_url %}
+        {% else %}
```

Comment:
Should we list category guides without external url?

Reply:
I think this was supposed to be linking to the GuidePage url_path

Reply:
yup. fixing